pkg/reconciler/namespace/namespace.go

```diff
@@ -56,8 +56,8 @@ const (
 
 	// Name of the corev1.Events emitted from the reconciliation process.
 	brokerCreated             = "BrokerCreated"
-	serviceAccountCreated     = "BrokerFilterServiceAccountCreated"
-	serviceAccountRBACCreated = "BrokerFilterServiceAccountRBACCreated"
+	serviceAccountCreated     = "BrokerServiceAccountCreated"
```

Comment:
might be better to have the 4 events: BrokerFilterXXX and BrokerIngressXXX, and you pass them as args when you call the method?
But I'm ok either way, if you want to leave it as is, it's fine.

Reply:
I guess with the message you are printing the name of the service account or role binding. We can leave it as is...